### PR TITLE
Remove Dynamo RaaS

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

[DYN-961](https://jira.autodesk.com/browse/DYN-961)
As a Dynamo dev, I want to remove DynamoRaaS nodes as part of DynamoCore and D4R

This PR removes DynamoRaaS dlls from D4R 2020. I am going to check if library has any left over.

Before: 1.3.3
image

After: 2.1.0
image

FYI: @AndyDu1985 @ZiyunShang This PR does not touch DynamoRevit install wix project, your team might consider removing that in the future. I believe the project there is out of date.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@mjkkirschner 

### FYIs

@alfarok @ColinDayOrg @ZiyunShang @AndyDu1985

### CODE REVIEW

File: src/DynamoRevit/DynamoRevit.cs [hunk 1]
```diff
@@ -593,8 +593,6 @@ private static void InitializeCore(DynamoRevitCommandData commandData)
         private static void InitializeAssemblies()
         {
             RevitAssemblyLoader.LoadAll();
-            AppDomain.CurrentDomain.AssemblyResolve +=
```
Comment (ColinDayOrg) [inline src/DynamoRevit/DynamoRevit.cs]: Is the RaaS the only analysis assembly that would cause this callback to be called?

**Comment by ColinDayOrg** [comment]
Just listed as an FYI on this PR, but other than my one question, LGTM.

File: src/DynamoRevit/DynamoRevit.cs (hunk 1 above)
Comment (QilongTang) [reply src/DynamoRevit/DynamoRevit.cs]: @ColinDayOrg  Good question. I believe so, the call of `Analyze.Render.AssemblyHelper.ResolveAssemblies` is actually in `DynamoRaaSHelper` which is the dll we are removing from the project by touching `DynamoRevit.csproj`. As the reference is removed, the actual code should not be called again.

**Comment by mjkkirschner** [comment]
LGTM - guess to test if it effects any builds we need to merge.

**Comment by QilongTang** [comment]
@ZiyunShang @AndyDu1985 This is a `mater` only change. I have built `master` and test locally, D4R should work fine and no longer delivers DynamoRaaS dlls. Since we no longer use installer for anything, wix project should be left along.  Can you test the Revit CI/CD with such change? Let me know if I should send a PR to bump D4R version. Merging.

**Pull request merged**